arcaflow_plugin_coremark_pro/coremark_pro_plugin.py

```diff
@@ -114,7 +117,7 @@ def certify_all(
         xcmd.append(f"-c{params.contexts}")
     if params.workers:
         xcmd.append(f"-w{params.workers}")
-    ca_cmd.append(f"XCMD={' '.join(xcmd)!r}")
+    ca_cmd.append(f"XCMD={' '.join(xcmd)}")
```

Comment:
Can you add this part (or a paraphrase of it) of your pull request description here (or near here) to explain why this is done?

>>> The make command called by subprocess to run the coremark-pro benchmarks uses an XCMD= set of sub-parameters that are passed on to the individual benchmarks. It turns out that quoting the string passed to XCMD= causes a parsing problem when those sub-parameters are passed to the benchmarks.

Reply:
I'm not sure what you are asking for. Do you want me to explain how the `!r` operative worked before it was removed from the code?

Reply:
Actually, I think I Just need your short explanation of why `XCMD` here.

> coremark-pro benchmarks use an XCMD= set of sub-parameters that are passed on to the individual benchmarks